### ISSUE TITLE
docsregen blocked

### ISSUE THREAD

**Issue by MarcAntoine-Arnaud** [cenotelie/cratery#26]
Using tag `b53f7c7041f5a7e1dd25ac459c4ea9740012482f` latest commit, I cannot call the `docsregen` endpoint.

It seems the database is locked.

**Comment by MarcAntoine-Arnaud** [comment]
after a restart of the docker container, I got:
```
The operation failed in the backend. (error returned from database: (code: 8) attempt to write a readonly database)
```

**Comment by woutersl** [comment]
Yes I am aware, I am fixing this.

**Comment by woutersl** [comment]
This should be fixed on main with [69e30f9](https://github.com/cenotelie/cratery/commit/69e30f95648a1321c21b5e122d2674a6fd05e95c)
